### ISSUE TITLE
Losing pyannote/whisper on template restart in Unraid

### ISSUE THREAD

**Issue by shadowsoze** [rishikanthc/Scriberr#34]
Howdy, so i can install everything easily using the updated template pointed to the nightly release, however if i restart the template and try and use it again, i'm getting this error:

```
DEV MODE -----> false
stderr: Traceback (most recent call last):
  File "/app/./diarize/local.py", line 2, in <module>
    from pyannote.audio import Pipeline
ModuleNotFoundError: No module named 'pyannote'
```

Not sure exactly why it's being deleted, it's almost as if nothing is being saved to the folders i define it to while setting up, because i can't find the whisper files either. 

![image](https://github.com/user-attachments/assets/019e1e13-1f71-4bfc-82a7-fa04c2e66b3f)

Any ideas?

**Comment by rishikanthc** [comment]
The nightly is behind. Use 0.3.0 that's the currently tested working one. Either ways what you're experiencing is weird. 
Could you share the docker compose you used please ?
When the app started did the configuration wizard run ?

**Comment by shadowsoze** [comment]
So looks like pointing to 0.3.0 instead of nightly fixed the issue...i've done a few restarts/stopping/starting of the docker template and functionality is retained with no issue, so i think i'm ok on that one. I also had to add PUID/PGID/UMASK info to the template but that's moreso on the template creator then the actual app so i'll let them know. 

The only issue i have now is diarization is hit or miss...but i think that's moreso of my source material (i've been testing with random youtube clips but they have background music so i'm assuming that's causing issues, i've done a few other voice files with just voice and it works well). 

Only other question i have is using LocalAI (https://localai.io/) and point to it using it's API instead of a seperate ollama process, in whether or not it's supported? I tried pointing to a llama model but it wasn't working for me, but i think the error was related to my hardware.
